src/subscribables/subscribable.js

```diff
@@ -75,6 +79,20 @@ var ko_subscribable_fn = {
                 ko.dependencyDetection.end(); // End suppressing dependency detection
             }
         }
+        // Notify subscribers to the ALL_EVENT event, but not if that was the event that was fired (prevent double events)
+        if (this.hasSubscriptionsForEvent(allEvent) && event != allEvent) {
```

Comment:
Because this _is_ the code for `notifySubscribers` - it's based entirely off of the existing implementation. Unless I'm missing something, calling again would result in an infinite loop